from pix_erase.application.errors.base import ApplicationError


class TaskNotFoundError(ApplicationError):
    ...
